app/Http/Client/Alchemy/AlchemyPendingRequest.php

```diff
@@ -321,6 +322,13 @@ public function parseNft(array $nft, int $networkId): Web3NftData
 
         $mintTimestamp = $this->getNftMintingDateProperty($nft);
 
+        $bannerImageUrl = Arr::get($nft, 'contractMetadata.openSea.bannerImageUrl');
+        if (! empty($bannerImageUrl)) {
+            $bannerImageUrl = NftImageUrl::get($bannerImageUrl, ImageSize::Banner);
+        } else {
+            $bannerImageUrl = null;
+        }
```

Comment:
We already have banners that make use of 1378 as image size before your changes, so how did those get used (see clickup for examples)?  or were those fetched by mnemonic? if so, might be worth it to change it there as well (+ moralis) for consistency so everything gets updated with similar banner sizes. Right now this solution solely applies to alchemy while there are other providers that can be used for banners as well